src/dmd/backend/var.d

```diff
@@ -147,68 +166,68 @@ int level = 0;                  /* declaration level                    */
                                 /* 2: function local declarations       */
                                 /* 3+: compound statement decls         */
 
-param_t *paramlst = NULL;       /* function parameter list              */
+//param_t *paramlst = null;       [> function parameter list              <]
 tym_t pointertype = TYnptr;     /* default data pointer type            */
 
 /************************
  * Bit masks
  */
 
-const unsigned mask[32] =
-        {1,2,4,8,16,32,64,128,256,512,1024,2048,4096,8192,16384,0x8000,
+const uint[32] mask =
+        [1,2,4,8,16,32,64,128,256,512,1024,2048,4096,8192,16384,0x8000,
          0x10000,0x20000,0x40000,0x80000,0x100000,0x200000,0x400000,0x800000,
          0x1000000,0x2000000,0x4000000,0x8000000,
-         0x10000000,0x20000000,0x40000000,0x80000000};
+         0x10000000,0x20000000,0x40000000,0x80000000];
 
-#if 0
-const unsigned long maskl[32] =
-        {1,2,4,8,0x10,0x20,0x40,0x80,
+static if(0)
+{
+const ulong[32] maskl =
+        [1,2,4,8,0x10,0x20,0x40,0x80,
          0x100,0x200,0x400,0x800,0x1000,0x2000,0x4000,0x8000,
          0x10000,0x20000,0x40000,0x80000,0x100000,0x200000,0x400000,0x800000,
          0x1000000,0x2000000,0x4000000,0x8000000,
-         0x10000000,0x20000000,0x40000000,0x80000000};
-#endif
+         0x10000000,0x20000000,0x40000000,0x80000000];
+}
 
 /* From util.c */
 
 /*****************************
  * SCxxxx types.
  */
 
-#if !SPP
-
-char sytab[SCMAX] =
+version (SPP) {} else
 {
-    #define X(a,b)      b,
-        ENUMSCMAC
-    #undef X
-};
 
-#endif /* !SPP */
-volatile int controlc_saw;              /* a control C was seen         */
-symtab_t globsym;               /* global symbol table                  */
+char[SCMAX] sytab;
```

Comment:
This is part of https://github.com/dlang/dmd/pull/8421 too, hence it's probably a good idea to merge https://github.com/dlang/dmd/pull/8421 first.